src/tests/ftest/daos_test/daos_core_test_dfs.yaml

```diff
@@ -11,11 +11,11 @@ hosts:
     - client-F
     - client-G
     - client-H
-
-timeout: 250
+timeout: 300
```

Comment:
FYI, since each of the tests, `test_u` and `test_p` have their own unique avocado test timeouts and the `daos_core_test_dfs.py` does not define any other test cases, this will never be used.